libsql/internal/http/hranaV2/hranaV2.go

```diff
@@ -286,7 +287,119 @@ func sendPipelineRequest(ctx context.Context, msg *hrana.PipelineRequest, url st
 	return result, false, nil
 }
 
+func (h *hranaV2Conn) executeMsg(ctx context.Context, query string, msg *hrana.PipelineRequest) (*hrana.PipelineResponse, error) {
```

Comment:
Can we remove `query` parameter from this function and delegate enriching error with query string to the caller?

Reply:
We can. Will it be any better? I guess it would keep error handling code better separated from the happy path but any other reason?

Reply:
It's just a little bit confusing that query is used only in the error message. If caller wants to enrich error message with more data - it have more information about additional properties and we better to delegate enrichment to the caller

Reply:
done